lib/drivers/animasDriver.js

```diff
@@ -1786,8 +1785,23 @@ module.exports = function (config) {
             progress(100);
             return cb(err, data);
           } else {
-            progress(100);
-            return cb(null, data);
+            progress(50);
+
+            cfg.api.upload.toPlatform(
+              data.post_dataservices,
```

Comment:
The two variables here aren't great, name-wise: `data.post_records` vs. `data.post_dataservices`... Perhaps you could come up with something clearer/more parallel to how you refer to the two APIs in api.js?